### DISCUSSION TITLE
TypeScript's `infer` keyword seemingly not being highlighted

### DISCUSSION THREAD

**Discussion by portasynthinca3** [github-linguist/linguist#5723] [Classification] [answered]
Example:
https://github.com/amogus-api/driver-ts/blob/cedd6eadab62ae71b949a5e77a0553bedcccfe61/src/common.ts#L40
![image](https://user-images.githubusercontent.com/14039901/148656422-726272e9-8c22-4990-99b8-53ef95e5e66d.png)

Both TS Playground and VSCode highlight `infer` as if it were a keyword:
![image](https://user-images.githubusercontent.com/14039901/148656274-71bb44bf-cf2a-40be-972d-15e5f29b8566.png)
![image](https://user-images.githubusercontent.com/14039901/148656394-1e0a85fe-589e-482e-8f4b-7c0b893f9d70.png)

**Comment by lildude** [comment, marked as answer]
As per the [troubleshooting doc](https://github.com/github/linguist/blob/master/docs/troubleshooting.md#theres-a-problem-with-the-syntax-highlighting-of-a-file) this is a problem with the grammar and not Linguist. Please raise this issue with the grammar maintainers.
